gitbase/templates/statefulset.yaml

```diff
@@ -29,13 +30,18 @@ spec:
       resources:
         requests:
           storage: "{{ .Values.storage.size }}"
+  {{- end }}
   template:
     metadata:
       labels:
         app: {{ template "gitbase.name" . }}
         release: {{ .Release.Name }}
     spec:
       volumes:
+      {{- if not .Values.storage.enablePVC }}
+      - name: gitbase-repos-volume
+{{ toYaml .Values.storage.customVolumesSpec | indent 8 }}
```

Comment:
Hmmm, what about
```
{{ toYaml required "Required storage.customVolumesSpec when storage.enablePVC is set to false" .Values.storage.customVolumesSpec | indent 8 }}

Reply:
That didn't work, found an alternative

Reply:
Too good to be true, but I like your alternative :-D